frontend/public/components/monitoring/alert-manager-config.tsx

```diff
@@ -74,36 +82,119 @@ const ReceiverTableHeader = () => {
       title: 'Routing Labels',
       props: { className: tableColumnClasses[2] },
     },
+    {
+      title: '',
+      props: { className: tableColumnClasses[3] },
+    },
   ];
 };
 ReceiverTableHeader.displayName = 'ReceiverTableHeader';
 
-const getIntegrationTypes = (receiver) => {
+const getIntegrationTypes = (receiver: AlertManagerReceiver): string[] => {
   /* Given receiver = {
        "name": "team-X-pager",
        "email_configs": [...],
        "pagerduty_configs": [...]
      };
-     returns "email, pagerduty"
+     returns ['email', 'pagerduty']
   */
   const integrationTypes = _.filter(_.keys(receiver), (key) => _.includes(key, '_configs'));
-  return _.join(_.map(integrationTypes, (type) => type.substr(0, type.indexOf('_configs'))), ', ');
+  return _.map(integrationTypes, (type) => type.substr(0, type.indexOf('_configs')));
```

Comment:
`_.filter` passes the key as the second param, so I think you can simplify this a bit. Also, there is a `_.endsWith` function.

`const integrationTypes = _.filter(receiver, (v, key) => _.endsWith(key, '_configs'));`

Reply:
fixed